profile/CSIP.xml

```diff
@@ -1393,7 +1393,7 @@
             xsi:schemaLocation="http://www.loc.gov/METS/ http://www.loc.gov/standards/mets/mets.xsd http://www.w3.org/1999/xlink http://www.loc.gov/standards/mets/xlink.xsd https://dilcis.eu/XML/METS/CSIPExtensionMETS https://dilcis.eu/XML/METS/CSIPExtensionMETS/DILCISExtensionMETS.xsd">
          </mets:mets>
     </Example>
-    <Example ID="metsRootElementExample3" LABEL="METS root element example of content with a type from the value list and representation content not in the value list">
+    <Example ID="metsRootElementExample3" LABEL="METS root element example of content with a type from the value list and representation content not in the value list instead a value decided upon inn the submission agreement is used">
```

Comment:
METS root element example of content with a type from the value list and representation content type is a custom value from the submission agreement.